src/core.ts

```diff
@@ -34,7 +34,7 @@ function generateResultsToUndo(
         if (!resultOfLint) throw new Error(`The result of ${resultToFix.filePath} is not found.`);
         // NOTE: THIS IS HACK.
         // Usually, the result with `output` property does not contain `source` property.
-        // However, `source` property is required for undoTransformation.
+        // However, `source` property is required for undofix.
```

Comment:
`undoTransformation` is the name of a method that was originally implemented in the Core class, but the method no longer exists (it was removed during the refactoring process).

It makes sense to change this sentence, but the name of the method named `undofix` does not exist as well. Could you please change it to the following?

```suggestion
        // However, `source` property is required to undo.
```

Reply:
@mizdra can you please check now.